### DISCUSSION TITLE
How to select multiple paths?

### DISCUSSION THREAD

**Discussion by tommy0831** [cypress-io/cypress#21158] [General]
How to select multiple paths using cy.get?  Is there an OR (|) locator like in webdriver?

in webdriver, one can do something like this:  //*[@data-t="id-1"] | //*[@data-t="id-2"]

Example:
[data-t="some-id-1"] | [data-t="some-id-2"]

https://www.w3schools.com/xml/xpath_syntax.asp

**Comment by jjhelguero** [comment]
[jQuery is bundled with Cypress](https://docs.cypress.io/guides/core-concepts/introduction-to-Cypress#Cypress-is-Like-jQuery) and can be used for many of its DOM commands. [jQuery allows multiple selectors](https://api.jquery.com/multiple-selector/), you'll have to use the `,` between each selector.

For your example to get both `data-t` elements
```
cy.get('[data-t=one],[data-t=two]')
```
